OrbitGl/ProcessData.h

```diff
@@ -5,13 +5,27 @@
 #ifndef ORBIT_GL_PROCESS_DATA_H_
 #define ORBIT_GL_PROCESS_DATA_H_
 
+#include <memory>
 #include <utility>
 
 #include "ModuleData.h"
 #include "OrbitBase/Logging.h"
+#include "OrbitBase/Result.h"
 #include "absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_set.h"
 #include "module.pb.h"
 #include "process.pb.h"
+#include "symbol.pb.h"
+
+// Small struct to model a space in memory occupied by a module.
+struct MemorySpace {
+  explicit MemorySpace(uint64_t start, uint64_t end) : start(start), end(end) {}
+  uint64_t start;
+  uint64_t end;
+  [[nodiscard]] std::string AddressRange() const {
```

Comment:
Can you name it something like `AddressRangeString` or `FormattedAddressRange`?

Reply:
done (FormattedAddressRange)